flo-runner/src/main/java/com/spotify/flo/context/ForkingContext.java

```diff
@@ -0,0 +1,112 @@
+/*-
+ * -\-\-
+ * Flo Runner
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.flo.context;
+
+import com.spotify.flo.EvalContext;
+import com.spotify.flo.Fn;
+import com.spotify.flo.TaskId;
+import com.spotify.flo.Tracing;
+import io.grpc.Context;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.util.Collections;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An {@link EvalContext} that runs tasks in sub-processes.
+ * <p>
+ * Forking can be disabled using the environment variable {@code FLO_DISABLE_FORKING=true}.
+ * <p>
+ * Forking is disabled by default when running in the debugger, but can be enabled by {@code FLO_FORCE_FORK=true}.
+ * <p>
+ * The basic idea here is to intercept the process fn by overriding {@link EvalContext#invokeProcessFn(TaskId, Fn)},
+ * and executing it in a sub-process JVM. The process fn closure and the result is transported in to and out of the
+ * sub-process using serialization.
+ */
+class ForkingContext extends ForwardingEvalContext {
+
+  private static final Logger log = LoggerFactory.getLogger(ForkingContext.class);
+
+  private ForkingContext(EvalContext delegate) {
+    super(delegate);
```

Comment:
Where did `transient` go?

Reply:
The `EvalContext` is no longer part of the process fn closure, so it won't be serialized.

https://github.com/spotify/flo/pull/89/files#diff-7605014c7b48e15103f2b401b5f8ccd5L134

Reply:
OK didn't realise that change. 👍